### PR TITLE
GNOME 42 support

### PR DESCRIPTION
**Pull request by ccat3z**
Register ClipboardIndicator using GObject.registerClass(). Fix https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/issues/342. It seems that ClipboardIndicator cannot be defined with Lang.Class since gnome shell 42. But I'm not sure why.

### CODE REVIEW

**Comment by TiagodePAlves** [comment]
I'm using this version now and it's working without problems. I've installed it with:

```bash
$ rm -rf $XDG_DATA_HOME/gnome-shell/extensions/clipboard-indicator@tudmotu.com
$ git clone -b gnome-42 git@github.com:ccat3z/gnome-shell-extension-clipboard-indicator.git $XDG_DATA_HOME/gnome-shell/extensions/clipboard-indicator@tudmotu.com
```

Testing with:

```bash
$ gnome-shell --version
GNOME Shell 42.0
```

**Comment by mlujann** [comment]
why was the branch deleted ?

File: extension.js [hunk 1]
```diff
@@ -62,8 +62,7 @@ class ClipboardIndicator extends PanelMenu.Button {
         this._clearLabelTimeout();
         this._clearDelayedSelectionTimeout();
 
-        // Call parent
-        this.parent();
+        super.destroy();
```
Comment (rmader) [inline extension.js:65]: Nit: typo in the commit message.

File: metadata.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
     "shell-version": [
         "40",
```
Comment (rmader) [inline metadata.json:3]: Did you ensure everything works with 40/41 as well?

**Review by rmader — APPROVED** [review]
Looks and works great for me on 42!

File: metadata.json (hunk 2 above)
Comment (Tudmotu) [reply metadata.json:3]: This is a good point. I will remove these, just in case. Changes like these are often not backward compatible with Gnome.

**Comment by Tudmotu** [comment]
Amazing!!
Thank you! :pray: 
Will merge it soon along with a couple others.

Apologies for taking so long, only now updated to 42 on my system.

**Pull request merged**